DependencyInjection/MonologExtension.php

```diff
@@ -468,6 +468,8 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['bubble'],
                 $handler['use_ssl'],
                 $handler['message_format'],
+                $handler['host'],
```

Comment:
Ok, I wasn't sure about that. If the php rfc for `default` had passed, I would pass that here (to keep the default value in monolog), but I can change it manually to the default value.

Should I do the same for the `api_version` parameter?